docs/multi-stage-query/security.md

```diff
@@ -60,17 +60,31 @@ Depending on what a user is trying to do, they might also need the following per
 
 
 
-## S3
+## Permissions for durable storage
 
-The MSQ task engine can use S3 to store intermediate files when running queries. This can increase its reliability but requires certain permissions in S3.
+The MSQ task engine can use Amazon S3 or Azure Blog Storage to store intermediate files when running queries. This can increase its reliability but requires certain permissions.
 These permissions are required if you configure durable storage. 
 
-Permissions for pushing and fetching intermediate stage results to and from S3:
+### S3
+
+The MSQ task engine needs the following permissions for pushing and fetching intermediate stage results to and from S3:
 
 - `s3:GetObject`
 - `s3:PutObject`
 - `s3:AbortMultipartUpload`
 
-Permissions for removing intermediate stage results:
+The MSQ task engine needs the following permissions for removing intermediate stage results:
+
+- `s3:DeleteObject`
```

Comment:
We should clump it with the permissions above, we should provide justification for each permission for consistency.